### DISCUSSION TITLE
Contribution 할 수 있는 방식이 어떤 게 있을까요?

### DISCUSSION THREAD

**Discussion by seastar105** [tony9402/baekjoon#49] [Question & Answer] [answered]
문제집에 넣을만한 문제 추천이나 코테 유형 알려드리는 것 외에 현재 받고 있는 게 어떤 것이 있나요?

예를 들자면 추천 문제의 소스파일이나 풀이 pdf의 PR 같은거요

**Comment by tony9402** [comment, marked as answer]
히익 시스타님 안녕하세요 ! <b>:fan:</b>
Contributor 조건에 대해 딱히 정해놓은게 아직 없긴 해용.

1. 코딩테스트에 나올만한 문제 또는 코딩테스트 대비로 풀어보면 좋은 문제 추천 (단, 레포에 없는 문제이어야 함.)
2. 최근 코딩테스트 유형 업데이트
3. Python, C++, Java 등 풀이 (PDF, 소스코드 등) 추가
4. 오타, 오류 지적 (또는 수정)

이외에도 이 레포에 조금이라도 도움이 된다면 최대한 넣어드리려고 하고 있어요 !
꼭 PR이 아니여도 Discussion, Issue 로 받고 있어요

**Reply by seastar105** [reply]
답변 감사합니다 레포 정말 멋져요!

**Reply by tony9402** [reply]
히익 감사합니다 !
